sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager.swift

```diff
@@ -387,7 +395,7 @@ internal final class URLSessionTransferManager: NSObject, TransferManager, URLSe
     }
 
     func pause(transfer: BlobTransfer) {
-        guard transfer.state.active else { return }
```

Comment:
Why this change? It doesn't seem like calling `pause` on a paused or failed transfer makes sense.

Reply:
Can't recall offhand. Switched it back to `.active`.